gradle-incremental-configuration-cache/src/main/java/com/palantir/gradle/configcache/incremental/IncrementalConfigurationCachePlugin.java

```diff
@@ -78,8 +78,13 @@ public final void apply(Project project) {
                     task.getLockFile().from(lockFilePath.toFile());
                 });
 
+        TaskProvider<DryRunConfigurationCacheEnabledTask> dryRunTask = project.getTasks()
+                .register("dryRunConfigurationCacheEnabledTasks", DryRunConfigurationCacheEnabledTask.class, task -> {
+                    task.getTasksToDryRun().set(new TaskListFile(targetTasksPath).loadTasks());
```

Comment:
This looks like we're reading a file at configuration time? Why not just pass the file in?